### ISSUE TITLE
Debugging through readonly internal module has weird reveal behaviour

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#13751]
Steps to Reproduce:

1. debug into a node module (e.g. process.send)
2. step over the implementation in that file

=> each step results in the current line to be centerd vertically which makes it hard to follow the code flow
=> I would only expect the scrolling to reveal the current line if the line is out of view

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/12438
